src/pages/MainPage/MainPage.js

```diff
@@ -1,14 +1,39 @@
 import { PageContainer } from '../../components/PageContainer/PageContainer'
 import SearchForm from '../../components/SearchForm/SearchForm'
+import SideScroller from '../../components/SideScoller/SideScroller'
+import styled from 'styled-components'
 import { useHistory } from 'react-router-dom'
+import useTrending from '../../hooks/useTrending'
 
 export default function MainPage() {
   const { push } = useHistory()
+  const {
+    trending: trendingScience,
+    isQuerying: isQueryingScience,
+  } = useTrending('science')
+  const {
+    trending: trendingTechnology,
+    isQuerying: isQueryingTechnology,
+  } = useTrending('technology')
 
   return (
-    <PageContainer>
+    <PageWrapper>
```

Comment:
PageContainer might be better here since wrapper mostly wrap around one child element while a container contains 1+ child elements (Read that some time ago while searching for a difference of wrapper & container)

Reply:
Naming wise you are correct but i have a styled compont ready which is called PageContainer. That im styling in each component to my needs.